src/mlpack/tests/cli_binding_test.cpp

```diff
@@ -19,7 +19,6 @@ using namespace std;
 using namespace mlpack;
 using namespace mlpack::bindings;
 using namespace mlpack::bindings::cli;
```

Comment:
Yeah, I guess I did not write it in the PR description, but I kept the `bindings::` namespace as-is, since it is not meant to be touched by users, and it makes sense to split each language into its own namespace (since many have functions that are the same name).